### ISSUE TITLE
Connection lost when idle even with "keep alive"

### ISSUE THREAD

**Issue by valepu** [voryx/ThruwayBundle#66]
I've already submitted this issue in this topic: https://github.com/voryx/ThruwayBundle/issues/46
But since i got no answer i wanted to make a new issue for this.

I launch the server from comand line and if i don't publish anything for some time (about 20-30 minutes) i get what looks like a log of all the previous calls (just like in the other issue) on the console and the server just hangs (the process is active, but it doesn't accept connections anymore).
Following the suggestion on the other issue i have made a topic where i publish every 5 minutes. Now it lasts some hours (about 6-7) but then the same thing happens eventually.

Is it possible to know what is causing the issue? I don't think a "keep alive" mechanism should be needed at all in the first place

I am working on a windows 10 machine using PHP 5.6 and dev-master version of the bundle (but it happened even with the latest release, and even with PHP 7.1)

**Comment by valepu** [comment]
So, it turns out that there were issues with reactphp/child-process on windows. Today i update all my dependencies and when I tried to run the command line for Thruway bundle it gave me an error that windows is not supported. I discovered that this was not caused by Thruway but by child-process. 
Here the issues tied to this change
https://github.com/reactphp/child-process/issues?utf8=%E2%9C%93&q=windows 
I read they completely removed windows support in the latest version since it was bugged to start with (which is what was causing my problems) but they are going to implement limited windows support in the next build.
Anyways as suggested in the readme for child-process project i have installed WSL and running Thruwaybundle command line from bash on Windows works just fine (been using it since 6-7 hours without ping or keep alive and a lot of idle time)
